task/09-functions-n-closures-tasks.js

```diff
@@ -26,7 +26,9 @@
  *
  */
 function getComposition(f,g) {
-    throw new Error('Not implemented');
+    return function(x) {
+        return f(g(x))
```

Comment:
Please, spaces, when nested call

Please, try to follow these rules https://javascript.info/coding-style

![Selection_008](https://user-images.githubusercontent.com/26328278/91710948-bd799780-eb8d-11ea-8ba0-f8d4cdba3efd.png)